### PR TITLE
protocol: include folder ID in i-team resolve result

### PR DESCRIPTION
**Pull request by strib**
KBFS will use this eventually.  It's ok for now if it remains unset until CORE-6635 is done.

Issue: KBFS-2598

### CODE REVIEW

**Comment by mlsteele** [comment]
Is this the thing that's going to be stored in the team sigchain?

File: protocol/avdl/keybase1/identify.avdl [hunk 1]
```diff
@@ -91,6 +91,7 @@ protocol identify {
     TeamID teamID;
     array<UserVersion> writers; // Members of the team that can write. The team OWNERs.
     map<UserVersion,IdentifyTrackBreaks> trackBreaks;
+    string folderID; // i-team TLF ID, if there is one
```
Comment (songgao) [inline protocol/avdl/keybase1/identify.avdl]: There's a `TLFID` defined in `common.avdl`. Is there a reason not to use that?

**Comment by songgao** [comment]
> Is this the thing that's going to be stored in the team sigchain?

Yeah I think so: https://keybase.atlassian.net/wiki/spaces/GEN/pages/113082369/Thoughts+on+Implicit+Teams+in+KBFS We are gonna call this when resolving a handle, and determine whether a TLF is backed by an i-Team by the response.

File: protocol/avdl/keybase1/identify.avdl (hunk 1 above)
Comment (strib) [reply protocol/avdl/keybase1/identify.avdl]: Sure, will do, thanks.

**Comment by strib** [comment]
@mlsteele: Yep, as per CORE-6635.

**Pull request merged**